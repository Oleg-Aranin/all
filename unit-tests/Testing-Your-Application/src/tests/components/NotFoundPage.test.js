import React from "react";
import {shallow} from 'enzyme'
import NotFoundPage from '../../components/NotFoundPage'
import {MemoryRouter} from "react-router-dom";


describe('NotFoundPage', () => {
    it('should render NotFoundPage correctly', () => {
        const wrapper = shallow(
            <MemoryRouter>
                <NotFoundPage/>
            </MemoryRouter>
        )

        expect(wrapper.html()).toMatchSnapshot()
    })


})
