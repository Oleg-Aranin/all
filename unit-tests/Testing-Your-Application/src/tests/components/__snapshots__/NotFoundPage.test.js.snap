// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseListItem should render ExpenseListItem correctly 1`] = `"<div><a href=\\"/edit/1\\"><h3>Gum</h3></a><p>195 - 0</p></div>"`;

exports[`NotFoundPage should render NotFoundPage correctly 1`] = `"<div>404 - <a href=\\"/\\">Go home</a></div>"`;
